velox/dwio/dwrf/test/ReaderTest.cpp

```diff
@@ -96,9 +91,24 @@ class TestReaderP
   std::unique_ptr<ExecutorBarrier> barrier_;
 };
 
+class TestReader : public testing::Test, public VectorTestBase {
+ protected:
+  std::vector<VectorPtr> createBatches(
+      const std::vector<std::vector<int32_t>>& integerValues) {
+    std::vector<VectorPtr> batches;
+    for (const auto& integerValue : integerValues) {
+      auto vector = makeFlatVector<int32_t>(integerValue);
+      auto rowVector = makeRowVector({vector});
+      batches.push_back(rowVector);
+    }
+    return batches;
+  }
+};
+class TestRowReaderPrefetch : public testing::Test, public VectorTestBase {};
```

Comment:
Add empty line here and before next class.

Reply:
Fixed